webapp/src/app/[lang]/page.tsx

```diff
@@ -46,8 +58,57 @@ export default function Home({ params }: { params: { lang: string } }) {
         Create Pull-Request
       </Button>
       {pullRequestUrl && <Link href={pullRequestUrl}> {pullRequestUrl} </Link>}
+      <p />
```

Comment:
What is the purpose of this? If it's to create a margin, add a margin to the underlying box instead, e.g. with this code which adds two units (I believe `rem` is the unit here) of margin to the top of the box (`mt` means "margin top"):

```tsx
<Box mt={2}>
```

I don't think you have to do anything about this now though, since any fix to #21 will replace this code.

Reply:
thank you
yes it was for creating a margin :) idk about that "mt"
anyway for sure as you mention next UI changes will do a lot of changes